src/store/state.js

```diff
@@ -2,8 +2,16 @@ export default
 {
     nearbyShops:
     [
-        { name: 'U Jadźki', distance: 300, street: 'Ketonowa 7', img: 'https://s-media-cache-ak0.pinimg.com/736x/a3/03/61/a30361985814a564e8b87515bb373661--shop-facade-coffeeshops.jpg' },
-        { name: 'U Józka', distance: 500, street: 'Betonowa 9', img: 'https://s-media-cache-ak0.pinimg.com/736x/a3/03/61/a30361985814a564e8b87515bb373661--shop-facade-coffeeshops.jpg' },
+        { id: 'idSetByServer1234567', name: 'U Jadźki', distance: 300, street: 'Ketonowa 7', img: 'https://s-media-cache-ak0.pinimg.com/736x/a3/03/61/a30361985814a564e8b87515bb373661--shop-facade-coffeeshops.jpg', location: [51.3668, -0.4628] },
```

Comment:
I'd consider storing shop's location as an object ```{latitude: 51.3668, longitude: -0.4628}```, rather than an array, that would be more descriptive.